manifests/o/Obsidian/Obsidian/1.8.7/Obsidian.Obsidian.installer.yaml

```
@@ -0,0 +1,57 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 1.8.7
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+Protocols:
+- obsidian
+FileExtensions:
+- md
+ReleaseDate: 2025-02-14
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.8.7/Obsidian-1.8.7.exe
+  InstallerSha256: 4301F8AB93D20FB513AB0D6E53011559C89C05E551731F2FC8DF320941299518
+  InstallerSwitches:
+    Custom: /currentuser
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.8.7/Obsidian-1.8.7.exe
+  InstallerSha256: 4301F8AB93D20FB513AB0D6E53011559C89C05E551731F2FC8DF320941299518
+  InstallerSwitches:
+    Custom: /allusers
+  ElevationRequirement: elevatesSelf
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.8.7/Obsidian-1.8.7.exe
+  InstallerSha256: 4301F8AB93D20FB513AB0D6E53011559C89C05E551731F2FC8DF320941299518
+  InstallerSwitches:
+    Custom: /currentuser
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.8.7/Obsidian-1.8.7.exe
+  InstallerSha256: 4301F8AB93D20FB513AB0D6E53011559C89C05E551731F2FC8DF320941299518
+  InstallerSwitches:
+    Custom: /allusers
+  ElevationRequirement: elevatesSelf
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.8.7/Obsidian-1.8.7.exe
+  InstallerSha256: 4301F8AB93D20FB513AB0D6E53011559C89C05E551731F2FC8DF320941299518
+  InstallerSwitches:
+    Custom: /currentuser
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.8.7/Obsidian-1.8.7.exe
+  InstallerSha256: 4301F8AB93D20FB513AB0D6E53011559C89C05E551731F2FC8DF320941299518

```

```suggestion
 InstallerSha256: 758DAAD483D6B90F66E6E773C46FC0505E5C34E74F764EB1BEA94EF02E002BB9
```

(Automated response - build 982.)